docs/manual/docs/user-guide/harvesting/harvesting-geonetwork-20.md

```diff
@@ -0,0 +1,27 @@
+# GeoNetwork 2.0 Harvester {#gn2_harvester}
```

Comment:
consider rename to harvesting-geonetwork-2x.md (to match pages names 3x and 4x below).

Reply:
Nevermind I see this is different